package.json

```diff
@@ -65,6 +65,8 @@
     "currency-formatter": "^1.4.2",
     "d3-selection": "^1.3.0",
     "d3-zoom": "^1.7.1",
+    "date-fns": "^2.23.0",
```

Comment:
Annoying to have both `date-fns` and `moment` imported... Perhaps we can look at transitioning our remaining moment calls to `date-fns`? Although that doesn't have to happen in this PR.

Reply:
We discussed that and agreed on phasing out moment.js in the technical call last week.